docs/platform/release-notes/tizen-6-5-m1.md

```diff
@@ -0,0 +1,554 @@
+  - A window resizing bug has been fixed.
+  - An image loading state bug has been fixed.
+  - Some TextField and TextEditor bugs have been fixed.
+
+
+### Multimedia framework
+
+#### New and changed features
+
+- Open source
+  - libvorbis
+    - Version 1.3.4 has been upgraded to version 1.3.7.
+  - libexif
+    - Version 0.6.21 has been upgraded to version 0.6.22.
+  - alsa-lib
+    - Version 1.0.28 has been upgraded to version 1.0.29.
+  - libjpeg
+    - Version 2.0.4 has been upgraded to version 2.0.6.
+  - giflib
+    - Version 5.1.9 has been upgraded to version 5.2.1.
+  - GraphicsMagick
+    - Version 1.3.34 has been upgraded to version 1.3.35.
+  - libav
+    - The libav has been removed and replaced to FFmpeg.
+  - FFmpeg
+    - Version 4.3.1 has been added.
+- Camera FW
+  - New HAL architecture has been applied.
+- Media tool
+  - New API set for referencing media packet has been added.
+- capi-media-vision
+  - Barcode Detection
+    - Three attributes for images rotation have been added.
+      - MV_BARCODE_DETECT_ATTR_ROTATION_DEGREES
+      - MV_BARCODE_DETECT_ATTR_ROTATION_COUNT
+      - MV_BARCODE_DETECT_ATTR_ROTATION_DIRECTION
+    - Enumeration for the direction
+      - MV_BARCODE_DETECT_ATTR_ROTATION_CLOCKWISE
+      - MV_BARCODE_DETECT_ATTR_ROTATION_COUNTER_CLOCKWISE
+      - MV_BARCODE_DETECT_ATTR_ROTATION_ALL
+    - One attribute for image enhancement has been added.
+      - MV_BARCODE_DETECT_ATTR_USE_ENHANCEMENT
+- Media Content
+  - New API for getting media handle by path has been added.
+- Image Util
+  - Support for decoding HEIF (High Efficiency Image Format) has been added.
+    - Decode HEIC brand of HEIF
+
+#### Fixes
+
+- Muse server
+  - Deadlock problem of signal handler has been fixed by calling malloc of async-signal-safe internally.
+- Resource Manager
+  - A conflict issue caused by not released resource of the abnormal terminated client has been fixed.
+- Media Streamer
+  - Custom sink node freeze bug has been fixed.
+  - An unexpected error problem with stop() API has been fixed.
+
+
+### Network and Connectivity
+
+#### New and changed features
+
+- User Awareness API
+  - The API set for device has been added.
+  - The API set for user has been added.
+  - The API set for service has been added.
+  - The API set for monitor has been added.
+  - Following API set is required the Bluetooth or network privilege as the device type.
+    - Privilege : http://tizen.org/privilege/bluetooth or http://tizen.org/privilege/network.get
+    - Affected API set
+      - ua_monitor_add_sensor
+      - ua_monitor_remove_sensor
+      - ua_user_add_device
+      - ua_user_remove_device
+      - ua_device_update
+      - ua_monitor_start_scan
+      - ua_monitor_stop_scan
+      - ua_monitor_start_presence_detection
+      - ua_monitor_stop_presence_detection
+      - ua_monitor_start_absence_detection
+      - ua_monitor_stop_absence_detection
+  - Following API is required the location privilege.

```

Comment:
Not clear 

What does following refer to ?

Reply:
Following API set requires the location privilege.

"Affected API set" are the following API set.

Reply:
@semun-lee This is better. 

Following API set requires the location privilege:

Reply:
will accept